### ISSUE TITLE
Unable to interact with Extensions list using touchscreen

### ISSUE THREAD

**Issue by sjetha** [Microsoft/vscode#9246]
- VSCode Version: Code 1.3.1 (e6b4afa53e9c0f54edef1673de9001e9f0f547ae, 2016-07-12T13:35:06.227Z)
- OS Version: Windows_NT ia32 10.0.14372

Steps to Reproduce:

1. Open the Extensions sidebar
2. Populate the search box with search criteria (such as '@popular')
3, Attempt to tap or scroll the results that appear in the extension sidebar

Expected result:
Items should scroll when interacted with, and when an item is tapped the right side should show details

Actual results:
Items do not scroll and tapping doesn't produce any results,
However the small "Enable" button does work.

**Comment by joaomoreno** [comment]
Duplicate of #9338 and already fixed in 1.4

**Comment by juliannojungle** [comment]
Can also confirm the bug in other touch devices, such as HP Detachable PC. The scroll was indeed fixed, but not tapping an item to see the details. Verified on VsCode version 1.11.2.

Workaround to see the extension details:
1 - Enable the full touch keyboard: On "Windows 10, All Settings, Devices, Typing", enable "Add the standard keyboard layout as a touch keyboard option";
2 - Open the touch keyboard, tap on the language icon (lower right) and select the full layout keyboard (this way you get the up/down arrows); 
3 - On VsCode, tap on the extensions icon, search for the desired extension, on the touch keyboard use the arrow down/up to navigate and press enter to view the details.

This way you can open the details panel, but sadly the touch scroll doesn't work on this panel as well...
